index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Abel Tefera | Software Engineer</title>
+  </head>
+  <body>
+    <header class="header">
+      <a id="logo" class="disable-default" href="#">Abel Tefera</a>
+      <nav class="hamburger">
+        <img src="assets/hamburger.svg" width="30" />
+      </nav>
+      <div class="links-container">
+        <a class="links disable-default" href="#">Portfolio</a>
+        <a class="links disable-default" href="#">About</a>
+        <a class="links disable-default" href="#">Contact</a>
+      </div>
+    </header>
+    <section class="container">
+      <h1 id="headline">I'm Abel, <br />Glad to see you!</h1>
+      <p id="description">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+
+      <h5 id="subtitle">LET'S CONNECT</h5>
+      <ul class="social-links">
+        <li class="disable-default">
+          <a class="disable-default">
+            <img src="assets/twitter.svg" />
+          </a>
+        </li>
+        <li class="disable-default">
+          <a class="disable-default">
+            <img src="assets/medium.svg" />
+          </a>
+        </li>
+        <li class="disable-default">
+          <a class="disable-default">
+            <img src="assets/angellist.svg" />
+          </a>
+        </li>
+        <li class="disable-default">
+          <a class="disable-default">
+            <img src="assets/github.svg" />
+          </a>
+        </li>
+      </ul>
```

Comment:
- [ ] You have done your best to ensure the use of [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) elements 👍 . Though you are close to making `img` more semantically correct. In this case, I suggest ensuring that the `img` tag has the [`alt text`](https://blog.hubspot.com/marketing/image-alt-text). Alt attributes enable screen readers to read the information about on-page images for the benefit of a person with a complete lack of sight, visually impaired, or who is otherwise unable to view the images on the page. Alt text will be displayed in place of an image if an image file cannot load 😞 .